CMakeLists.txt

```diff
@@ -33,4 +33,4 @@ install(FILES bmi/bmi.h bmi/bmi_cem.h DESTINATION include/cem)
 install(TARGETS bmi_cem DESTINATION lib)
 install(TARGETS bmi_cem-static DESTINATION lib)
 
-add_subdirectory(testing)
+add_subdirectory(testing)
```

Comment:
You removed the trailing whitespace. Your version is missing an end-of-line character.